packages/ui/core/Button/__tests__/__snapshots__/Button.test.tsx.snap

```diff
@@ -1,19 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Button renders correctly 1`] = `
-<TouchableOpacity
-  activeOpacity={0.2}
-  gradient={true}
-  onPress={[MockFunction onPress]}
-  rounded={true}
-  size="small"
+<View
```

Comment:
Why is this not an `TouchableOpacity` anymore?

Reply:
It was made automatically using `yarn jest -u` to update the snapshots. This change is related to this issue https://github.com/callstack/react-native-testing-library/issues/450